modules/nodes-scheduler-node-affinity-configuring-required.adoc

```diff
@@ -31,34 +31,47 @@ metadata:
   name: <node_name>
   labels:
     e2e-az-name: e2e-az1
+#...
 ----
 ====
 
-. In the `Pod` spec, use the `nodeAffinity` stanza to configure the `requiredDuringSchedulingIgnoredDuringExecution` parameter:
+. Create a pod with a specific label in the pod spec:
 +
-.. Specify the key and values that must be met. If you want the new pod to be scheduled on the node you edited, use the same `key` and `value` parameters as the label in the node.
+.. Create a YAML file with the following content:
 +
-.. Specify an `operator`. The operator can be `In`, `NotIn`, `Exists`, `DoesNotExist`, `Lt`, or `Gt`. For example, use the operator `In` to require the label to be in the node:
+[NOTE]
+====
+You cannot add an affinity directly to an existing scheduled pod.
+====
 +
 .Example output
 [source,yaml]
 ----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s1
 spec:
-  affinity:
+  affinity: <1>
     nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
+      requiredDuringSchedulingIgnoredDuringExecution: <2>
         nodeSelectorTerms:
         - matchExpressions:
-          - key: e2e-az-name
-            operator: In
+          - key: e2e-az-name <3>
             values:
             - e2e-az1
             - e2e-az2
+            operator: In <4>
+#...
 ----
+<1> Adds a pod affinity.
+<2> Configures the `requiredDuringSchedulingIgnoredDuringExecution` parameter.
+<3> Specifies the `key` and `value` that must be met. If you want the new pod to be scheduled on the node you edited, use the same `key` and `value` parameters as the label in the node.
```

Comment:
In a couple of other files in this PR, I see similar text that mentions "the `key` and `values` that must be met". Is `value` correct here, or should it be `values`?